Swat/SwatTreeNode.php

```diff
@@ -206,14 +170,12 @@ public function key()
      *               {@link SwatTreeNode} object. If the next child node is
      *               invalid, false is returned.
      */
+    #[ReturnTypeWillChange]
     public function next()
     {
         return next($this->children);
```

Comment:
I think we looked at this one and the return value was used in one place in code. We could/should fix that and make this void.